x-pack/plugins/actions/server/actions_client.ts

```diff
@@ -155,6 +155,7 @@ export class ActionsClient {
       {
         actionTypeId,
         name,
+        isMissingSecrets: true,
```

Comment:
Now that this is `isMissingSecrets`, it should be set to `false` on create and update to reflect the variable meaning

Reply:
Ops, you are right. Naming is hard :-)